app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :events
```

Comment:
#### Associations

##### User as event creator

* [ ] User model: `has_many :events, foreign_key: :creator_id` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

- Even though the Events table uses the user_id attribute as the foreign key automatically you were asked to use the creator_id alias instead so let's make the changes to make it possible.